webapp/components/_new/generic/CounterIcon/CounterIcon.vue

```diff
@@ -1,4 +1,4 @@
-<template>
+<template v-show="cappedCount > 0">
   <span class="counter-icon">
     <base-icon :name="icon" />
     <span :class="counterClass">{{ cappedCount }}</span>
```

Comment:
Unfortunately this does not work, because the `v-show` is set at the `template`tag and because `cappedCount` is no number but a string.
And if the first would work then the whole icon would vanish and not only the number in the red circle.

Therefore I suggest:

```suggestion
<template>
  <span class="counter-icon">
    <base-icon :name="icon" />
    <span v-if="count > 0" :class="counterClass">{{ cappedCount }}</span>
```